python/essex-config/essex_config/sources/env_source.py

```diff
@@ -0,0 +1,18 @@
+"""Environment variables source class implementation."""
+
+import os
+from typing import Any
+
+from essex_config.sources.source import Source
+
+
+class EnvSource(Source):
+    """Class to get the configuration from the environment."""
+
+    def get_data(self) -> dict[str, Any]:
+        """Get the data dictionary."""
+        return {**os.environ}
```

Comment:
Is this expensive? I would expect for us to only read from the environ what is required from it.